source/image-handler/package.json

```diff
@@ -23,7 +23,8 @@
     "build:init": "rm -rf package-lock.json && rm -rf dist && rm -rf node_modules",
     "build:zip": "zip -rq image-handler.zip . -x template.yml",
     "build:dist": "mkdir dist && mv image-handler.zip dist/",
-    "build": "npm run build:init && npm install --production && npm run build:zip && npm run build:dist"
+    "build": "npm run build:init && npm install --production && npm run build:zip && npm run build:dist",
+    "dev": "sam build --use-container; sam local start-api -p 8181"
```

Comment:
I tried it but it didn't work as I expected out of the box.
It was trying to create a new lambda rather than updating the existing one.

I am sure that's just some config to tweak, but I didn't dig much.